platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusConfig.java

```diff
@@ -36,5 +36,5 @@
 @ConfigData("consensus")
 public record ConsensusConfig(
         @ConfigProperty(defaultValue = "26") int roundsNonAncient,
-        @ConfigProperty(defaultValue = "1500") int roundsExpired,
```

Comment:
how is this related to the health monitor?

Reply:
It's not, but I was asked to make the change.

Reply:
I address this with Lazar offline